Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -527,6 +527,16 @@ public let CodeGenerators: [CodeGenerator] = [
         let value = b.randVar()
         b.storeToScope(value, as: b.genPropertyNameForWrite())
     },
+
+    CodeGenerator("EvalGenerator") { b in
+        if !b.context.contains(.codeString) {
```

Comment:
Is this because the lifter currently can't deal with nested codestrings (i.e. JS template literals)? I think you can already get nested codestrings e.g. from splice mutations (which insert snippets of code into the current program), so maybe the lifter anyway needs to be able to deal with nested codestrings?

Reply:
Yes that's correct and you're also right that the splice mutator could insert instructions that could generate nested codestrings. Either we have the lifter detect nested codestrings and escape backticks correctly or add a check in the splice mutator to prevent splicing code string instructions. I haven't fully evaluated the impact the two approaches would have, do you have a preference on approach?

Reply:
Having given this some more thought, it's better to handle nested codestrings in the lifter that it would be to restrict splicing mutations. I've added/reverted some changes to support nested codestrings.